README.md

```diff
@@ -41,7 +41,14 @@ StanUpMan's UI is built with [svelte](svelte.dev/) and source is hosted at [stan
 
 ## Setting UP
 
-# Contributors
+# Support
+
+Reach out to our maintainer on [element](https://matrix.to/#/#standupman:matrix.org) or contact us at support@standupman.xyz
+
+# Contributing
+
+Feel free to [open an issue](https://github.com/standupman/standupman/issues), create new features, fix bugs and/or improving documentation.
```

Comment:
Some few corrections...

Feel free to "open issues" (not an issue), submit patches to fix bugs and or improve documentation. Please discuss new features with the maintainers in the contributors room.

Reply:
Will do!